app/controllers/photos_controller.rb

```diff
@@ -13,18 +13,22 @@ def show
   end
 
   def index
+    where = {}
     if params[:crop_slug]
       @crop = Crop.find params[:crop_slug]
-      @photos = Photo.by_crop(@crop)
+      where = { crops: @crop.id }
     elsif params[:planting_id]
       @planting = Planting.find params[:planting_id]
-      @photos = @planting.photos
-    else
-      @photos = Photo.all
+      where = { planting_id: @planting.id }
     end
-    @photos = @photos.order(created_at: :desc)
-      .includes(:owner)
-      .paginate(page: params[:page])
+
+    @photos = Photo.search(
+      load:     false,
+      boost_by: [:created_at],
+      where:    where,
```

Comment:
I was wondering, why is the search done by `id` if we get the `slug` for searching all of these entities - (Crop, Planting, Member, etc)? 
One call to the DB could be saved if  ES would store slug instead of id (or both).
What am I missing?